docs/docs/concepts/architecture/solution.md

```diff
@@ -1,67 +1,60 @@
-Make sure to read our [Production Guide](/self-hosting/manage/production#prefer-postgresql) before you decide to use it.
+Depending on your projects needs our general recommendation is to run Zitadel across multiple availability zones in the same region or across multiple regions.
+Alternatively you can run Zitadel with Postgres which is [Enterprise Supported](/docs/support/software-release-cycles-support#partially-supported).
+Make sure to read our [Production Guide](/docs/self-hosting/manage/production#prefer-postgresql) before you decide to use it.
+Consult the [Postgres documentation](https://www.postgresql.org/docs/) for more details.
 
 ## Scalability
 
-ZITADEL can be scaled in a linear fashion in multiple dimensions.
+Zitadel can be scaled in a linear fashion in multiple dimensions.
 
 - Vertical on your compute infrastructure
 - Horizontal in a region
 - Horizontal in multiple regions
 
 Our customers can reuse the same already known binary or container and scale it across multiple server, data center and regions.
 To distribute traffic an already existing proxy infrastructure can be reused. 
-Simply steer traffic by path, hostname, IP address or any other metadata to the ZITADEL of your choice.
+Simply steer traffic by path, hostname, IP address or any other metadata to the Zitadel of your choice.
 
-> To improve your service quality we recommend steering traffic by path to different ZITADEL deployments
+> To improve your service quality we recommend steering traffic by path to different Zitadel deployments
 > Feel free to [contact us](https://zitadel.com/contact/) for details
 
 ## Example Deployment Architecture
 
 ### Single Cluster / Region
 
-A ZITADEL Cluster is a highly available IAM system with each component critical for serving traffic laid out at least three times.
-As our storage layer (CockroachDB) relies on Raft, it is recommended to operate odd numbers of storage nodes to prevent "split brain" problems.
-Hence our reference design for Kubernetes is to have three application nodes and three storage nodes.
+A Zitadel Cluster is a highly available IAM system with each component critical for serving traffic laid out at least three times.
+Our storage layer (Postgres) is built for single region deployments.
+Hence our reference design for Kubernetes is to have three application nodes and one storage node.
 
-> If you are using a serverless offering like Google Cloud Run you can scale ZITADEL from 0 to 1000 Pods without the need of deploying the node across multiple availability zones.
+> If you are using a serverless offering like Google Cloud Run you can scale Zitadel from 0 to 1000 Pods without the need of deploying the node across multiple availability zones.
 
-:::info
-CockroachDB needs to be configured with locality flags to proper distribute data over the zones
-:::
-
-![Cluster Architecture](/img/zitadel_cluster_architecture.png)
+![Cluster Architecture](/img/zitadel_cluster_architecture_v3.png)
```

Comment:
as far as i can see, that's the only time we use the image, can't we just replace the image itself as well?

Reply:
replaced